packages/react-examples/src/react-charting/VerticalBarChart/VerticalBarChart.Basic.Example.tsx

```diff
@@ -197,6 +197,9 @@ export class VerticalBarChartBasicExample extends React.Component<IVerticalBarCh
             culture={window.navigator.language}
             chartTitle="Vertical bar chart basic example "
             data={points}
+            xAxisTitle="Huge x axis title which is very long and will be truncated with ellipsis at the end
+            of the text and will be shown on hover of the text"
+            yAxisTitle="Huge y axis title which is very long and will be truncated with ellipsis"
```

Comment:
Add a new example with moderate length titles. Do not add showXAxisLablesTooltip to it. Revert the change in basic example

Reply:
done